packages/perseus/src/widgets/expression/expression.util.test.ts

```diff
@@ -0,0 +1,23 @@
+import {expressionItem2} from "./expression.testdata";
+import getExpressionPublicWidgetOptions from "./expression.util";
+
+import type {PerseusExpressionWidgetOptions} from "@khanacademy/perseus-core";
+
+describe("getExpressionPublicWidgetOptions", () => {
+    it("should return the correct public options without any answer data", () => {
+        // Arrange
+        const options: PerseusExpressionWidgetOptions =
+            expressionItem2.question.widgets["expression 1"].options;
```

Comment:
It looks like `visibleLabel` and `ariaLabel` are not included in the test data? I'd prefer to define the input inline rather than relying on shared data from `expression.testdata`.

Sharing data between tests makes them harder to understand (you have to look at multiple files to understand the test setup) and prone to false passes if the test data changes later.

```suggestion
        const options: PerseusExpressionWidgetOptions = {
            buttonSets: ["basic"],
            functions: ["f", "g", "h"],
            times: false,
            visibleLabel: "the visible label",
            ariaLabel: "the aria label",
            buttonsVisible: "always",
        };
```

Reply:
I think this makes sense. I know Ben advocates the builder method and I like the generator method, but either way it's nice to have a shared logic to generate an object that can just be overwritten for the parts that are most relevant to the test. For instance we have `generateTestExpressionWidget` in `test-utils.ts` which can give you a skeleton that you can then modify without risking changes to other tests.

Reply:
Is the previously created test data just something to ignore moving forward then? When should it be used? Will update to one of the above methods for sure though 😅

Reply:
Updated :)